src/alert.cpp

```diff
@@ -244,8 +246,22 @@ bool CAlert::ProcessAlert(const std::vector<unsigned char>& alertKey, bool fThre
     return true;
 }
 
```

Comment:
Using zero as the key sounds a bit sketchy, as we would likely get the second error erasing or replacing the first. Perhaps you could hash the strMessage and use the hash instead of zero? Or was overwriting the messages intended behavior? Or is there something I'm missing?